src/algorithm/shoelace_formula.rs

```diff
@@ -0,0 +1,14 @@
+use num_traits::Float;
+use types::LineString;
+
+pub fn twice_area<T>(linestring: &LineString<T>) -> T where T: Float {
+    if linestring.0.is_empty() || linestring.0.len() == 1 {
+        return T::zero();
+    }
+    let mut tmp = T::zero();
+    for ps in linestring.0.windows(2) {
+        tmp = tmp + (ps[0].x() * ps[1].y() - ps[1].x() * ps[0].y());
```

Comment:
@mbattifarano we're also already computing it in [convexhull.rs](https://github.com/georust/rust-geo/blob/master/src/algorithm/convexhull.rs#L51) although in a slightly different way, and it's used in a couple of places in the [Polygon Distance PR](https://github.com/georust/rust-geo/pull/115/files#diff-af7ca56bf0fcfc5aced02a9a325eecc6R713), so it's probably worth thinking about how to unify them.